doc/Release_Build_Creation.rst

```diff
@@ -1,16 +1,35 @@
-=======================
-Release Build Creation
-=======================
+=========================================================
+Compile & Build Complete Cortx Stack Via Docker Container
+=========================================================
 
 This file consists of the procedure that should be followed to generate the release build outside the Seagate network using `cortx-build <https://github.com/orgs/Seagate/packages/container/package/cortx-build>`_ docker image. 
 
-***************
-Procedure
-***************
+*****************************************
+Specifications to setup Virtual Machine
+*****************************************
 
-#. Setup a `CentOS 7.8.2003 <http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/>`_ system.
+- Single-Node VM deployment
 
-   - You can use a Virtual Machine (VM) also.
+  - Please create VM(s) with at least 1 cores (vCPU) and 4GB of RAM.
```

Comment:
This is our requirement. Deployment fails if user is using less then 4GB RAM. My request to remove the word "please". 
Kindly update the sentence to:

- Create a [CentOS 7.8.2003 VM](http://isoredirect.centos.org/centos/7.8.2003/isos/x86_64/) with the following configuration:
  - RAM: 4GB
  - CPU: 1 core
  - RAW disks: 2
      - Data Disk: 1 (Capacity: 20GB)
      - Metadata Disk: 1 (Capacity: 10% of total Data Disk Size)
  - NIC: 3
- Ensure IP’s have assigned to all NICs. For this deployment interface name is considered as eth33, eth34, and eth35.

You can also remove the note if you are writing as suggested above

Reply:
done